src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -140,16 +156,54 @@ const EditShiftHours = ({
           setShowEditShiftHours(false);
         }
       },
+      true,
       handleServerValidationErrors
     );
   };
 
+  const getCombinedErrors = () => {
```

Comment:
Consider lifting this function out of here with its own tests/test file. EditShiftHours seems to be doing a lot and this concern could be separated out.

Reply:
👍 done this now, though now sure where it should live? I've put it in utils for now, interested in your thoughts!

Reply:
Looks great and I like it and where you have put it